src/Razor/src/Microsoft.VisualStudio.RazorExtension/Microsoft.VisualStudio.RazorExtension.csproj

```diff
@@ -277,4 +277,10 @@ using Microsoft.VisualStudio.Shell;
       <InformationalVersion>$(VsixVersion)</InformationalVersion>
     </PropertyGroup>
   </Target>
+
+  <Target Name="_RemoveDuplicateSourceGenerator" AfterTargets="ResolvePackageAssets" Condition="'$(TargetFramework)' == '$(DefaultNetCoreTargetFramework)'">
```

Comment:
This target name is misleading. The issue isn't that the generator is "duplicate". But the issue is that it targets Roslyn v4.0, which only works with VS 2022.

Also, do you need to remove the TargetFramework condition? This project builds for `net472`.

Reply:
```suggestion
  <!-- The JsonSourceGenerator only works on VS 2022, and we are still building on VS 2019. Remove it for now. -->
  <Target Name="_RemoveJsonSourceGenerator" AfterTargets="ResolvePackageAssets">
```

Reply:
@pranavkm - you should be able to commit the above suggestion and the build should start passing (I can't since I don't have write permissions).